opendigitaleducation/entcore
# [ODE] Notification for files upload

# Description

Push a notification every time a file is uploaded.

Note : A stress test is coming.

## Fixes

WB-1544

## Type of change

Please check options that are relevant.


## Which packages changed?

Please check the name of the package you changed


## Tests

1. With user A, go to conversation app
2. Create a new message
3. Attach a file to it
4. Send it
5. Check the logs that the file was analyzed

# Reminder

- Security flaws
- Performance impacts (think bulk !)
- Unit tests were replayed
- Unit tests were added and/or changed
- I have updated the reminder for the version including my modifications
